saleor/graphql/product/tests/test_product_discounted_price.py

```diff
@@ -412,4 +413,5 @@ def test_sale_remove_catalogues_updates_products_discounted_prices(
         product_ids=[product.pk],
         category_ids=[category.pk],
         collection_ids=[collection.pk],
+        variant_ids=[],
```

Comment:
We should update this test to check if this mock receives `variant_id` when passed to mutation.

Reply:
Test updated.